server/api/order.js

```diff
@@ -4,9 +4,59 @@ const {
 } = require("../db");
 module.exports = router;
 
+//get a user's existing cart items
```

Comment:
this needs to be broken up into two routes. First do the get. If it returns empty then call the post route to create frp, axois. This will keep our API RESTFUL.

We will aim to fix this in a future ticket